src/Index.js

```diff
@@ -652,12 +652,109 @@ Index.prototype.getSettings = function(callback) {
   var indexObj = this;
   return this.as._jsonRequest({
     method: 'GET',
-    url: '/1/indexes/' + encodeURIComponent(indexObj.indexName) + '/settings',
+    url: '/1/indexes/' + encodeURIComponent(indexObj.indexName) + '/settings?getVersion=2',
```

Comment:
This might "break" some implementations. Any client that uses getSettings to get synonyms in some sync scripts won't have access to them anymore, so they might trigger unnecessary index build by sending synonyms again.

I think by default `getVersion` should be omitted and there should be an option to set it. Something like `getSettings({ version: 2 })`?